parsers/imageAltTag.js

```diff
@@ -0,0 +1,47 @@
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('parser:imageAltTag');
+var parse = require('./lib/parse');
+
+/*
+ * Extract alt text from img tags
+ */
+
+var PARSER_COMPLETED_SUCCESSFULLY = false;
+var FACEBOOK_ALT_TEXT_START = 'Image may contain:';
+
+function parseAltContent(altText) {
+    var content = altText.split(FACEBOOK_ALT_TEXT_START)[1];
+    var parsedWords = content.split(',');
+    parsedWords = _.map(parsedWords, function (e) { return e.trim() });
+    return parsedWords;
+}
+
+function implementation(snippet) {
+
+    debug("→ %j", _.omit(snippet, ['html']));
+
+    var $ = cheerio.load(snippet.html);
+    var returnValue = [];
+
+    var altTagFounds = $('img[alt^="'+FACEBOOK_ALT_TEXT_START+'"]');
+
+    _.each(altTagFounds, function (e) {
+        returnValue.push(parseAltContent(e.attribs.alt));
+    })
+    returnValue = _.flatten(returnValue);
+
+    PARSER_COMPLETED_SUCCESSFULLY = true;
+    return {
+        'imageAltTag': returnValue
+    }
+};
+
+return parse.please({
+    'name': 'imageAltTag',
+    'requirements': { 'imageAltTag': true },
```

Comment:
This can't be a `requirements`, because the imageAltTag is set by this parser itself. The requirements are intended for the parser to process only the posts that might contain useful content for the plugin. In this case, would be 

`requirements: { type: 'photo' }`